src/docs/tips.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Advanced Tips
+---
+
+# Advanced Tips
+
+In this section we'll outline some advanced hints and tips to get the most out of developing with Eclipse Theia.
+
+## Upgrading functionality in a VS Code Extension for use in Theia
+
+It's possible to declare a custom Theia API namespace which can be implemented in a standard VS Code Extension.
```

Comment:
Theia allows running VS Code extension by providing a compatible API (see [this overview](https://theia-ide.org/docs/extensions/) for details). It is possible to extend this API to allow VS Code extensions running in Theia to access additional functionality compared to when they run within VS Code.